tests/test_electricity_storage.py

```diff
@@ -98,3 +99,12 @@ def test_source_sink(self):
             power_bat_network * (1 - is_charging) / discharge_eff,
             atol=tol,
         )
+
+        # effective power discharge variable should always be the bigger or equal to the negative
+        # of the charge variable and bigger then zero. The zero should only occur if battery is
+        # charging. When a goal would be set to minimise discharge it should match the charge power,
+        # however now this goal is not turned on.

```

Comment:
What is the point of this "however now this goal is not turned on."? Do you mean this is not used yet, but will be used in future, with test (Add TODO if this is the intention) to follow as well?

Reply:
Added TODO